VRDR.CLI/Program.cs

```diff
@@ -1101,6 +1104,32 @@ record = coding.DeathRecord;
                 Console.WriteLine(payload);
                 return 0;
             }
+            else if (args.Length == 2 && args[0] == "filter")
+            {
+                Console.WriteLine($"Filtering file {args[1]}");
+
+                BaseMessage baseMessage = BaseMessage.Parse(File.ReadAllText(args[1]));
+                
+                FilterService FilterService = new FilterService("./VRDR.Filter/NCHSIJEFilter.json", "./VRDR.Filter/IJEToFHIRMapping.json");
+
+                string filteredFile;
+                
+                try
+                {
+                    filteredFile = FilterService.filterMessage(baseMessage).ToJson();
+                    BaseMessage.Parse(filteredFile);
+                    Console.WriteLine($"File successfully filtered");
+                    
+                    string filteredFilePath = "./VRDR.Filter/filteredFile.json";
+                    File.WriteAllText(filteredFilePath, filteredFile);

```

Comment:
Please make the filtered file output a configurable file in args[2] so that it doesn't overwrite sections of the source code tree.